centos-7.template

```diff
@@ -5,13 +5,14 @@ selinux --disabled
 
 # Use network installation
 url --url="http://mirror.centos.org/centos/7/os/x86_64/"
-network --bootproto=dhcp --device=link --activate --onboot=on
+network --bootproto=dhcp --device=eth0 --activate --onboot=on
+network --bootproto=dhcp --device=eth1 --activate --onboot=on
```

Comment:
We require eth1 because in case of virtualbox eth0 is taken by internal IP "10.0.2.15" which is not able to communicate outside so eth1 will have IP for external communication.